source/access/how_to_request_more_quota.rst

```diff
@@ -11,9 +11,32 @@ option depends on data storage policies of the site managing your VSC account,
 VO or Tier-1 project as well as on current capacity of the storage system.
 
 Before requesting more storage, please check carefully the
-:ref:`current data usage of your VSC account <quota>` and identify which
+:ref:`current data usage of your VSC account <checking disk usage>` and identify which
 file system needs a larger quota.
 
+.. _more quota inode:
+
+Requesting more inode quota
+===========================
+
+If you need more :ref:`inode quota <quota>`, you should first try to
+reduce the number of files as much as possible:
+
+- Clean up the files you no longer need.
+- Pack the remainder in a tarball, e.g. ``tar zcf myfiles.tgz file1 <...>
```

Comment:
most useful comment here would be to use a parallel zip instead of default, since it is by far the slowest part of this comment and most usefull when doing large cleanups

Reply:
something like this? `tar -I pigz cf myfiles.tgz file1 <...> `

Reply:
yes. we have pigz on all our nodes, so that is fine with me

Reply:
hm, it's not available in the leuven genius login

Reply:
ok, they just installed it.
antwerpen also has it.